pom.xml

```diff
@@ -38,7 +38,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <ddf.version>2.11.0-SNAPSHOT</ddf.version>
-        <ddf.support.version>2.3.6</ddf.support.version>
+        <ddf.support.version>2.3.7-SNAPSHOT</ddf.support.version>
```

Comment:
Don't know if there was discussion on this, but should we be on the snapshot version of ddf support?  Is it more appropriate to release 2.3.7?